samples/cpp/ngraph_function_creation_sample/main.cpp

```diff
@@ -249,156 +221,118 @@ std::shared_ptr<ov::Function> createNgraphFunction() {
 int main(int argc, char* argv[]) {
     try {
         // -------- Get OpenVINO runtime version --------
-        slog::info << "OpenVINO Runtime: " << ov::get_openvino_version() << slog::endl;
+        slog::info << *ov::get_openvino_version() << slog::endl;
 
         // -------- Parsing and validation of input arguments --------
-        if (!ParseAndCheckCommandLine(argc, argv)) {
-            return EXIT_SUCCESS;
+        if (argc != 3) {
+            std::cout << "Usage : " << argv[0] << " <path_to_lenet_weights> <device>" << std::endl;
+            return EXIT_FAILURE;
         }
-
-        // -------- Read input --------
-        std::vector<std::string> images;
-        parseInputFilesArguments(images);
-        OPENVINO_ASSERT(!images.empty(), "No suitable images were found");
+        const std::string weights_path{argv[1]};
+        const std::string device_name{argv[2]};
 
         // -------- Step 1. Initialize OpenVINO Runtime Core object --------
-        slog::info << "Loading OpenVINO runtime" << slog::endl;
         runtime::Core core;
 
         slog::info << "Device info: " << slog::endl;
-        std::cout << core.get_versions(FLAGS_d) << std::endl;
+        slog::info << core.get_versions(device_name) << slog::endl;
 
         // -------- Step 2. Create network using ov::Function --------
+        slog::info << "Create model from weights: " << weights_path << slog::endl;
+        std::shared_ptr<ov::Function> model = createNgraphFunction(weights_path);
+        printInputAndOutputsInfo(*model);
+
+        OPENVINO_ASSERT(model->inputs().size() == 1, "Incorrect number of inputs for LeNet");
+        OPENVINO_ASSERT(model->outputs().size() == 1, "Incorrect number of outputs for LeNet");
+
```

Comment:
probably we can remove these `asserts` which are a kind of junk here?

Reply:
idea to show that model input/outputs should be validated, to ensure we got appropriate model